com.ktechpit.ultimate-media-downloader.yaml

```diff
@@ -0,0 +1,225 @@
+          type: html
+          url: https://waf.io/
+          version-pattern: href="waf-(\d[\d.-]*)\"
+          url-template: https://waf.io/waf-$version
+    post-install:
+      - mkdir -p /app/etc/mpv/
+      # save screenshots at ~/Pictures/mpv-screenshots
+      - echo "screenshot-directory=~/Pictures/UMD4-MPV-screenshots" > /app/etc/mpv/mpv.conf
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share
+      
+      
+  - name: ffmpeg
+    config-opts:
+      - '--disable-debug'
+      - '--disable-doc'
+      - '--enable-rpath'
+      - '--enable-gpl'
+      - '--disable-static'
+      - '--enable-shared'
+      - '--enable-libfontconfig'
+      - '--enable-libfreetype'
+      - '--enable-libopus'
+      - '--enable-libpulse'
+      - '--enable-librsvg'
+      - '--enable-libvpx'
+      - '--enable-libdav1d'
+      - '--enable-libxml2'
+      - '--enable-libx264'
+      - '--enable-libx265'
+      - '--enable-libv4l2'
+      - '--enable-libmp3lame'
+      - '--enable-libass'
+      - '--enable-gnutls'
+      - '--enable-libvorbis'
+      - '--enable-libaom'
+      - '--enable-libopenjpeg'
+    cleanup:
+      - /share/ffmpeg/examples
+    sources:
+      - type: archive
+        url: https://www.ffmpeg.org/releases/ffmpeg-4.4.1.tar.xz
+        sha256: eadbad9e9ab30b25f5520fbfde99fae4a92a1ae3c0257a8d68569a4651e30e02
+        
+
+  - name: ultimate-media-downloader
+    buildsystem: simple
+    build-commands:
+      - unsquashfs -no-progress mqCImuZBihW5Od6DMGOZ3BQqwLN9i3LW_25.snap
+      - rm -f mqCImuZBihW5Od6DMGOZ3BQqwLN9i3LW_25.snap
+      
+      - mkdir -p ${FLATPAK_DEST}/share/metainfo
+      - mkdir -p ${FLATPAK_DEST}/share/application
+      - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/
+      
+      - mv squashfs-root/usr/bin/ultimate-media-downloader ${FLATPAK_DEST}/bin/ultimate-media-downloader
+      - mv squashfs-root/usr/lib/* ${FLATPAK_DEST}/lib/
+      - mv squashfs-root/meta/gui/icon.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/com.ktechpit.ultimate-media-downloader.png
+      - mv squashfs-root/usr/share/applications/ultimate-media-downloader.desktop ${FLATPAK_DEST}/share/application/com.ktechpit.ultimate-media-downloader.desktop
+      
+      - desktop-file-edit --set-icon=${FLATPAK_ID} ${FLATPAK_DEST}/share/application/com.ktechpit.ultimate-media-downloader.desktop
+      
+      - desktop-file-edit --set-comment="Download Media from Internet" ${FLATPAK_DEST}/share/application/com.ktechpit.ultimate-media-downloader.desktop
+      
+      - install -Dm644 com.ktechpit.ultimate-media-downloader.metainfo.xml ${FLATPAK_DEST}/share/metainfo/com.ktechpit.ultimate-media-downloader.metainfo.xml
+      
+      - ls -l
+      
+      - cp -rf hicolor/* ${FLATPAK_DEST}/share/icons/hicolor/
+      - rm -r squashfs-root
+    sources:
+      - type: file
+        path: dist/linux/com.ktechpit.ultimate-media-downloader.metainfo.xml
+      - type: dir
+        path: dist/linux/
```

Comment:
the snap come without icons??????

Reply:
I just use one Icon there.

Reply:
as long as it is 128x128 or bigger it's all you need.

Reply:
Got it thanks :) I added them to avoid any build failures. Will remove in my once the build will be stable.